pkgs/kona/src/cli/coverage/diff_addition_filter_merge.ts

```diff
@@ -105,9 +105,11 @@ function removeLineChange(
     index: number,
     allChanges: Change[]
 ) {
-    const chunkChanges = allChanges.slice(index - change.ln + 1, index + 1)
+    const chunkChanges = allChanges.slice(0, index + 1)
```

Comment:
We want to get all the changes up to and including this deletion. The previous value did not capture all the changes.